xml/decoders/SoundTraxx_Tsu2_Diesel.xml

```diff
@@ -220,7 +232,10 @@
             <model model="TSU-4400 Baldwin Diesels" numOuts="6" numFns="30" maxMotorCurrent="4A" connector="other" productID="295" comment="Model 885020">
                 <size length="69" width="30.5" height="14" units="mm"/>
             </model>
-            <model model="TSU-PNP EMD Diesels" numOuts="6" numFns="30" maxMotorCurrent="2A" connector="DropIn" productID="288" comment="Model 885013">
+	    <model model="TSU-KN1 Diesels" numOuts="4" numFns="30" maxMotorCurrent="5mA" connector="DropIn" productID="373" comment="Model 885032">
```

Comment:
Is this change intended? It seems that you remove one decoder model, `Model 885013`, and replaces that with another, `Model 885032`.

Reply:
My fork shows 885013 to be present along  with the add of 885032 I had done at line 235. 
So this pushed 885013 down to 238. 

What happened ?  Michael's changes were not committed to Master while I made changes to my fork ?

One question.. How Do I update my FORK so it is current ?

Reply:
> What happened ? Michael's changes were not committed to Master while I made changes to my fork ?

@mmosher5501 looked at this PR and confirmed that it was OK. So I have merged this PR into master.

> One question.. How Do I update my FORK so it is current ?

You merge upstream/master into your master.

Which tool do you use for git? Command line git, GitHub Desktop or something else?

Reply:
Direct in Github JMRI.. I created a fork of the master and upload changes to it and then issue a PR.  But I question if the FORK is updated or if I must update it somehow before making changes.  I see messages that my fork is xx commits behind.  

Example is this incident here. Michael's changes were not reflected in my FORK,  so changes i did collided with his.  

He had done changes to the include statements, grouping several within the CV.  My fork did not have these changes in it. hence the confusion.

Reply:
Which OS do you use on your computer? Windows, Linux or Mac? Or a mobile phone?

If you are using Windows or Mac, I recommend that you download GitHub Desktop. It's easy to use for beginners and I have used it for a couple of years and it works very well.
https://desktop.github.com/

A tutorial for GitHub Desktop and JMRI is here:
https://www.jmri.org/help/en/html/doc/Technical/githubdesktopinto.shtml

Using GitHub Desktop, it's easy to keep your fork updated.

Reply:
Windows  and I used the Github desktop for a time. If I can refresh the fork via desktop, I should be able to do same online.. just can't find how, short of erase and refork.

Reply:
> If I can refresh the fork via desktop, I should be able to do same online..

I'm not sure that's possible. What you need to do is to merge upstream/master into your master. And I don't think GitHub has added that feature to their web site. I might be wrong, but I haven't seen it.